lib/editor/tunnel.js

```diff
@@ -81,8 +81,10 @@ class EditorTunnel {
                         // This is the initial connect request.
                         const localWSEndpoint = `${thisTunnel.localWSProtocol}://127.0.0.1:${thisTunnel.port}/device-editor${request.url}`
                         debug(`Connecting local comms to ${localWSEndpoint}`)
-
-                        const tunnelledWSClient = newWsConnection(localWSEndpoint, { rejectUnauthorized: false })
+                        const options = {
+                            [thisTunnel.localWSProtocol]: { rejectUnauthorized: false }
+                        }
+                        const tunnelledWSClient = newWsConnection(localWSEndpoint, options)
```

Comment:
What has driven this change? Reading the docs for [`ws`](https://github.com/websockets/ws/blob/master/doc/ws.md#new-websocketaddress-protocols-options) I cannot find anything that says option should be nested like this. I can only find linked examples that provide `rejectUnauthorized` as a top-level option.

Reply:
Because `options.rejectUnauthorized` fills the log with ` DeprecationWarning: Got: "options.rejectUnauthorized" is now deprecated, please use "options.https.rejectUnauthorized"
(Use `node --trace-deprecation ...` to show where the warning was created)`

To be fair, this was tenuous fit under the title of "improve logs"

Reply:
ok, got overzelous with moving `rejectUnauthorized` to `.https.rejectUnauthorized` - this is a mistake. update incoming.

Reply:
reverted in [9d9b421](https://github.com/flowforge/flowforge-device-agent/pull/128/commits/9d9b4210b9a6176ded446883ecf34173a2a95ef5)